src/target/target_probe.h

```diff
@@ -98,5 +98,6 @@ bool stm32mp15_cm4_probe(target_s *target);
 bool zynq7_probe(target_s *target);
 
 void lpc55_dp_prepare(adiv5_debug_port_s *dp);
+int nrf91_dp_prepare(adiv5_debug_port_s *const dp);
```

Comment:
`const` here will be ignored by the compiler. This should be a `clang-tidy` lint. Having it in the function definitions is correct though.